# This contains the message from the GPS
bool fix
int16 NumSat
float64 latitude # Deg
float64 longitude # Deg
float64 altitude # m
float64 speed # m/s
float64 ground_course # rad
float64 covariance # m
